frag-nav/src/main/java/com/ncapdevi/fragnav/FragNavController.kt

```diff
@@ -277,7 +285,11 @@ class FragNavController constructor(private val fragmentManger: FragmentManager,
                 } else {
                     fragment = getRootFragment(currentStackIndex)
                     // Handle special case of indexes, restore tag of removed fragment
-                    var tag = fragment.tag ?: fragmentStacksTags[index].peek()
+                    var tag = fragment.tag
+                    // TODO: Better empty stack handling
```

Comment:
This one was throwing StackEmptyException in NavDrawer example (NavDrawer example > Open NavDrawer > Click any menu) and in tests.